ts/parts/Utilities.ts

```diff
@@ -2641,6 +2654,54 @@ const getStyle = H.getStyle = function (
     return style;
 };
 
+/**
+ * Get the defer as a number value from options set in the annotations or
+ * in the stackLabels object. If not defined inherit from the plotOptions.series
+ *
+ * @function Highcharts.getDeferTime
+ *
+ * @param {Highcharts.Chart} chart
+ *        The chart instance.
+ *
+ * @param {number | boolean} defer
+ *        Defer defined in the stackLabels/annotations options
+ *
+ * @return {number}
+ *         The numeric value.
+ */
+const getDeferTime = H.getDeferTime = function (
```

Comment:
I think a bit of refactoring is needed here.
* Is this function supposed to mimick the current behaviour of data labels? In this case it should also be used by data labels.
* It currently only looks at `plotOptions`-level options, but what if the user changes the `defer` or `duration` settings of the series item? Probably a general Utilities.ts function is not the way to go, it should rather be a `Series` method. For stack labels it can simply do an array.reduce over the stack series and use the highest `deferTime` reported by the series.
* In the last condition it looks for `plotOptions.line.animation` which can't be right for other series types... But all this will hopefully sort out correctly if we move the log to the Series object and only look for `this.options.animation`.

Reply:
@TorsteinHonsi

Your proposal to rewrite it as a `Series` method totally makes sense for the `stackLabels` and the `dataLabels` - I agree. But what about the `annotation`? If `annotation.animation.defer` will be enabled without any number value - where the defer value should come from? From the `plotOptions` as it is now?

Reply:
I think in that case it should be reduced from `chart.series`, the same way as the defer from stack labels is reduced from `yAxis.series`.